<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="floor0CallBtn">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>100</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Call elevator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor1CallBtn">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>150</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Call elevator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor2CallBtn">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>200</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Call elevator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor3CallBtn">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>250</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Call elevator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor4CallBtn">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>300</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Call elevator</string>
    </property>
   </widget>
   <widget class="QLabel" name="floor0Lbl">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="floor1Lbl">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="floor2Lbl">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="floor3Lbl">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="floor4Lbl">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>300</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="goToFloorLbl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Go to floor</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoFloor0Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoFloor1Btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoFloor2Btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoFloor4Btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoFloor3Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConnectionLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Connection:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConnectionStatusLbl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>165</red>
          <green>29</green>
          <blue>45</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>165</red>
          <green>29</green>
          <blue>45</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
